projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/yes-no/read-yes-no-field.component.ts

```diff
@@ -4,7 +4,7 @@ import { YesNoService } from './yes-no.service';
 
 @Component({
   selector: 'ccd-read-yes-no-field',
-  template: `<span class="text-16">{{formattedValue ? (caseField.label | rpxTranslate : null : formattedValue) : null}}</span>`
+  template: `<span class="text-16">{{formattedValue | rpxTranslate}}</span>`
```

Comment:
This will disable the welsh should be formattedValue ? formatted | rpxTranslate : null : formattedValue) : null